src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -122,6 +122,7 @@ public static bool IsPrivilegedProcess
 
         public static bool IsThreadingSupported => (!IsWasi && !IsBrowser) || IsWasmThreadingSupported;
         public static bool IsWasmThreadingSupported => IsBrowser && IsEnvironmentVariableTrue("IsBrowserThreadingSupported");
+        public static bool IsNotWasmThreadingSupported => !IsWasmThreadingSupported;
```

Comment:
You probably did it to make `WasmThreadingSupported` search match both variables, but can we consider `IsWasmThreadingUnsupported` ot `IsWasmThreadingNotSupported`? They make me less confused when reading